tool/tele/cli/register.go

```diff
@@ -33,38 +33,38 @@ func RegisterCommands(app *kingpin.Application) Application {
+	tele.BuildCmd.SetDeps = loc.LocatorSlice(tele.BuildCmd.Flag("set-dep", "Rewrite dependencies section in the image manifest file during vendoring, e.g. 'gravitational.io/site-app:0.0.39' will overwrite dependency to 'gravitational.io/site-app:0.0.39'.").Hidden())
+	tele.BuildCmd.SkipVersionCheck = tele.BuildCmd.Flag("skip-version-check", "Skip version compatibility check.").Hidden().Bool()
+	tele.BuildCmd.Parallel = tele.BuildCmd.Flag("parallel", "Specifies the number of concurrent tasks. If < 0, the number of tasks is not restricted, if unspecified, then tasks are capped at the number of logical CPU cores.").Int()
+	tele.BuildCmd.Quiet = tele.BuildCmd.Flag("quiet", "Suppress any extra output to stdout.").Short('q').Bool()
 
-	tele.ListCmd.CmdClause = app.Command("ls", "Display a list of user applications published in remote Gravity Hub")
-	tele.ListCmd.Runtimes = tele.ListCmd.Flag("runtimes", "Show only runtimes").Short('r').Hidden().Bool()
-	tele.ListCmd.Format = common.Format(tele.ListCmd.Flag("format", fmt.Sprintf("Output format, one of: %v", constants.OutputFormats)).Default(string(constants.EncodingText)))
-	tele.ListCmd.All = tele.ListCmd.Flag("all", "Display all available versions").Bool()
+	tele.ListCmd.CmdClause = app.Command("ls", "Display a list of cluster and application images published to Gravity Hub.")
+	tele.ListCmd.Runtimes = tele.ListCmd.Flag("runtimes", "Show only runtimes.").Short('r').Hidden().Bool()
+	tele.ListCmd.Format = common.Format(tele.ListCmd.Flag("format", fmt.Sprintf("Output format: %v.", constants.OutputFormats)).Default(string(constants.EncodingText)))
+	tele.ListCmd.All = tele.ListCmd.Flag("all", "Display all available versions.").Bool()
 
-	tele.PullCmd.CmdClause = app.Command("pull", "Pull an application from remote Gravity Hub")
-	tele.PullCmd.App = tele.PullCmd.Arg("app", "Name of application to download: <name>:<version> or just <name> to download the latest").Required().String()
-	tele.PullCmd.OutFile = tele.PullCmd.Flag("output", "Name of downloaded tarball, defaults to <name>-<version>.tar").Short('o').String()
-	tele.PullCmd.Force = tele.PullCmd.Flag("force", "Overwrite existing tarball").Short('f').Bool()
-	tele.PullCmd.Quiet = tele.PullCmd.Flag("quiet", "Suppress any extra output to stdout").Short('q').Bool()
+	tele.PullCmd.CmdClause = app.Command("pull", "Pull a cluster or application image from Gravity Hub.")
+	tele.PullCmd.App = tele.PullCmd.Arg("image", "Cluster or application image to download: <name>:<version> or just <name> to download the latest.").Required().String()
+	tele.PullCmd.OutFile = tele.PullCmd.Flag("output", "File name of the downloaded image. Defaults to <name>-<version>.tar.").Short('o').String()
+	tele.PullCmd.Force = tele.PullCmd.Flag("force", "Overwrite the existing image file if it already exists.").Short('f').Bool()
```

Comment:
one ~~existing~~
Also, sometimes this flag is called `overwrite` - should be either consistently `overwrite` or consistently `force`